src/components/pagination/pagination.twig

```diff
@@ -28,7 +28,12 @@
           </a>
         {% else %}
           {# Fallback for non-link pages like ellipsis gaps for first/last pages #}
-          <span class="c-pagination__action">
+          <span
+            class="c-pagination__action"
+            {% if page.title == '&hellip;' or page.title == '…' %}
+              aria-hidden="true"
```

Comment:
What happens to the list element semantically when this inner span is hidden? Would it be better to hide the `li`?

Reply:
Oh, oops, good call!

Reply:
[Done!](https://github.com/cloudfour/cloudfour.com-patterns/pull/1983/commits/6eafa6b2e016dd1dc691ac9ec25f67599d3b3e07)